src/js/control-bar/seek-to-live.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * @file live-display.js
+ */
+import Button from '../button';
+import Component from '../component';
+import * as Dom from '../utils/dom.js';
+
+/**
+ * Displays the live indicator when duration is Infinity.
+ *
+ * @extends Component
+ */
+class SeekToLive extends Button {
```

Comment:
there's probably no need and it will overly complicate things, but should this extend LiveDisplay?

Reply:
I don't think that it should, as it isn't a button right now.